services/sidecar/rollup/rpc/bridge/bridge_client.go

```diff
@@ -62,6 +63,14 @@ func (c *BridgeClient) GetRequiredStakeAmount(ctx context.Context) (*big.Int, er
 	return c.IRollup.CurrentRequiredStake(&bind.CallOpts{Pending: false, Context: ctx})
 }
 
+func (c *BridgeClient) RequireFirstUnresolvedAssertionIsRejectable(ctx context.Context, address common.Address) error {
+	return c.IRollup.RequireFirstUnresolvedAssertionIsRejectable(&bind.CallOpts{Pending: false, Context: ctx}, address)
+}
+
+func (c *BridgeClient) RejectFirstUnresolvedAssertion(ctx context.Context, address common.Address) (*types.Transaction, error) {
+	return c.IRollup.RejectFirstUnresolvedAssertion(&bind.TransactOpts{Context: ctx}, address)
+}
```

Comment:
use `TxManager` instead for tx submission (this struct should only be used for `view` functions) to handle reorgs/resubmission etc